src/xmipp/libraries/reconstruction/resolution_monogenic_signal.h

```diff
@@ -101,6 +103,7 @@ class ProgMonogenicSignalRes : public XmippProgram
 	bool halfMapsGiven;
 	Image<double> Vfiltered, VresolutionFiltered;
 	Matrix1D<double> freq_fourier;
+	Matrix1D<double> freq_fourier_x, freq_fourier_y, freq_fourier_z;
```

Comment:
these don't look like they should be public

Reply:
then, let's convert them in private